cmd/cyphernetes/main_test.go

```diff
@@ -92,7 +92,9 @@ Flags:
 Global Flags:
   -A, --all-namespaces     Query all namespaces
   -l, --loglevel string    The log level to use (debug, info, warn, error, fatal, panic) (default "info")
-  -n, --namespace string   The namespace to query against (default "default")`
+  -n, --namespace string   The namespace to query against (default "default")
+      --no-color           Turn off color formatting
+  `
```

Comment:
the line break here is redundant (output remains the same) and I think inconsistent